packages/sdk-core/src/model/http/model/submit/request/BacktraceSubmitMemory.ts

```diff
@@ -0,0 +1,48 @@
+export interface DataBacktraceSubmitMemory {
```

Comment:
how about renaming it to BacktraceDataSubmitMemory ? Sounds more natural and even the file name suggest it should start this way

Reply:
renamed to `BacktraceSubmitDataMemory`